lr_face/data.py

```diff
@@ -18,6 +20,20 @@
 Augmenter = Callable[[np.ndarray], np.ndarray]
 
 
+class Yaw(Enum):
+    FRONTAL = "straight"
+    TURNED = "slightly_turned"
```

Comment:
toch wel enigszins verwarrend - turned en sideways klinken een beetje als hetzelfde

mssn frontal, half_turned en profle, naar analogie van pitch?

Reply:
Zal ik aanpassen, dat is inderdaad duidelijker.